edgelet/edgelet-core/src/identity.rs

```diff
@@ -4,6 +4,7 @@ use std::fmt;
 
 use failure::Fail;
 use futures::Future;
+use serde_derive::*;
```

Comment:
Import `{Deserialize, Serialize}` specifically please. Glob imports are an upgrade hazard. Here and other places.